src/services/userService.js

```diff
@@ -0,0 +1,67 @@
+import {config} from '../config/config';
+
+
+export const userService = {
+    register,
+    login,
+    logout
+
+};
+
+function login(email,password) {
+    const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({email,password}),
+    };
+    return fetch(`${config.apiUrl}/login`, requestOptions)
+    .then(handleResponse)
+    .then(user => {
+        // login successful if there's a jwt token in the response
+        //if (user.token) {
+            // store user details and jwt token in local storage to keep user logged in between page refreshes
+            localStorage.setItem('user', JSON.stringify(user));
+        //}
+        return user;
+    },
+    (error)=>{
+        return Promise.reject(error);
+    }
+);
+}
+function logout() {
+    localStorage.removeItem("user");
+}
+function register(user) {
+    const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
```

Comment:
You have written this method and almost equal for login. Isn't it reasonable to write more general methods for GET, POST, PUT, DELETE request with params, for example: data, url and so on?